ledger-tool/src/main.rs

```diff
@@ -1902,6 +1915,16 @@ fn main() {
                         debug!("Feature gate deactivated: {address}");
                     }
 
+                    for address in feature_gates_to_activate {
+                        let mut account =
+                            AccountSharedData::new(1, Feature::size_of(), &feature::id());
```

Comment:
well, i think updating feature gates should be avoided for mb.

iirc, `feature_gates_to_deactivate` is intended for dev/testnet purpose.

capitalization will need to be adjusted, which isn't good thing for mb... also, i'd recommend some extensive testing...

also, `1` isn't enough, i think it needs to rent-exempt.

Reply:
I have similar concerns to Ryo. It would be nice if we could designate a payer to fund the feature-account's rent-exempt reserve (like what happens during a normal activation) to avoid any capitalization adjustments, but it also feels like an overreach for ledger-tool to be able to do that debit.

Reply:
Yea. I agree on the tests, for sure we need to test it better.

Reply:
Do you know if we update capitalization when we deactivate a feature? I didnt' see a code path for that.

Reply:
> Do you know if we update capitalization when we deactivate a feature? I didnt' see a code path for that.

Yes, looks like the capitalization gets reset based on all account removals here: https://github.com/solana-labs/solana/blob/19454bf56efb41cebd354a942d3b63235541292a/ledger-tool/src/main.rs#L2027
I think this will implicitly handle a capitalization bump for a feature add as well. But it's still not great for mnb.

Reply:
This happens every time we create a snapshot for a restart anyway, right? It's just that we usually don't provide arguments that would affect the capitalization. We should probably print a warning on how the capitalization differs from the original snapshot actually.

Reply:
> We should probably print a warning on how the capitalization differs from the original snapshot actually.

Agreed! We should add that regardless.